packages/connect/src/api/cardano/cardanoOutputs.ts

```diff
@@ -43,3 +57,40 @@ export const transformOutput = (output: any): OutputWithTokens => {
 
     return result;
 };
+
+export const sendOutput = async (typedCall: any, outputWithData: OutputWithData) => {
+    const MAX_CHUNK_SIZE = 512;
```

Comment:
I wouldn't convert to bytes before sending, see [comment](https://github.com/vacuumlabs/trezor-suite/pull/1#discussion_r907136861) above.

Perhaps this would be better?
```suggestion
    const MAX_CHUNK_SIZE = 256 * 2; // 256 hex encoded bytes
```